map.go

```diff
@@ -492,7 +492,13 @@ func (m *Map) nextKey(key interface{}, nextKeyOut internal.Pointer) error {
 // It's not possible to guarantee that all keys in a map will be
 // returned if there are concurrent modifications to the map.
 func (m *Map) Iterate() *MapIterator {
```

Comment:
Can we add a comment here saying that calling `Iterate()` won't work for Kernels < 4.4.132 and that as a workaround you can call `IterateFrom` with a non-existing key?

Reply:
Done.